www/js/survey/multilabel/multi-label-ui.js

```diff
@@ -535,8 +515,7 @@ angular.module('emission.survey.multilabel.buttons',
       Logger.log("trip starting at "+trip.start_fmt_time+": executing recompute");
       trip.waitingForMod = false;
       trip.timeoutPromise = undefined;
-      console.log("Recomputing display trips on ", viewScope);
-      viewScope.recomputeDisplayTimelineEntries();
+      $rootScope.broadcast("recomputeListEntries");
```

Comment:
When moving dependencies, I found that the instance of jQuery from npm didn't have a `parents()` function which caused `findViewElement()` to fail. (Maybe a versioning thing)
I started troubleshooting that, but I decided to see what the ultimate purpose of that function was anyway because it felt weird that we need jQuery for something in the multilabel buttons.

I realized it was all a complex way of getting the main view to update from the buttons, when we're not sure which main view we're in. Now that we don't have the diary screen, there's only one view this applies to: the infinite scroll list. So this can all be greatly simplified, and in the process we can remove a little bit of jQuery from the world

I definitely should have added a note explaining that, and I definitely want to test this behavior in-depth. Hard to keep track of large swooping changes like this

Reply:
addressed in #980 - I fixed this up, as it did have some issues
It was this commit: https://github.com/e-mission/e-mission-phone/pull/980/commits/812d854a1575eff0e8e1c21efa5ee3e1cee834a8

Reply:
that commit has the fix resizing the list elements *and* the fix for `recomputeListEntries`. And the fix for `recomputeListEntries` seems to only be to use `$broadcast` instead of `broadcast`. Is that right?

Reply:
All correct - but now since everything has been converted to React components, we don't use broadcasts; we use functions that are passed down and allow us to update the trips list in LabelTab